Module 2

MiguelAlvarezHernanz opened the pull request:
Module 2: EDA

Contains: two .ipynb files (+ files for downloading data) with my approach to the work proposed for Module 2.

MiguelAlvarezHernanz committed:
new file:   src/module_1/module_1_extra_fun.py
	modified:   src/module_1/module_1_meteo_api.py
	modified:   tests/module_1/test_meteo_api.py

MiguelAlvarezHernanz committed:
modified:   src/module_1/module_1_extra_fun.py
	modified:   src/module_1/module_1_meteo_api.py
	modified:   tests/module_1/test_meteo_api.py

MiguelAlvarezHernanz committed:
new file:   data/groceries/abandoned_carts.parquet
	new file:   data/groceries/inventory.parquet
	new file:   data/groceries/orders.parquet
	new file:   data/groceries/regulars.parquet
	new file:   data/groceries/sampled_box_builder_df.csv
	new file:   data/groceries/users.parquet
	modified:   poetry.lock
	modified:   pyproject.toml
	new file:   src/module_2/EDA_1.ipynb
	new file:   src/module_2/EDA_2.ipynb
	new file:   src/module_2/__init__.py
	new file:   src/module_2/download_data.py
	new file:   src/module_2/download_data2.py

MiguelAlvarezHernanz committed:
modified:   .gitignore
	modified:   src/module_2/EDA_1.ipynb
	modified:   src/module_2/EDA_2.ipynb

MiguelAlvarezHernanz committed:
modified:   src/module_2/EDA_1.ipynb

MiguelAlvarezHernanz committed:
modified:   src/module_2/EDA_2.ipynb

RafaZ7 commented on data/groceries/abandoned_carts.parquet:
Cuidado con esto! Me imagino que ya lo sabes, pero no debes añadir estos fichero al repositorio. Aunque has metido la ruta de `data` en el `.gitignore`, probablmente ya los tenían en el repositorio antes de meterla.

MiguelAlvarezHernanz replied:
Así es, supongo que me di cuenta de meterlo en el .gitignore después del primer push y luego se me olvidó eliminarlos. Gracias!

RafaZ7 commented on src/module_2/EDA_1.ipynb:
¿Puedes subirme una versión de los notebooks en markdown? En el notebook es un poco complicado hacer la revisión. Gracias!

MiguelAlvarezHernanz committed:
Removed directory 'data' from Git tracking

MiguelAlvarezHernanz committed:
modified:   poetry.lock
	modified:   pyproject.toml
	new file:   src/module_2/EDA_1.md
	new file:   src/module_2/EDA_1_files/EDA_1_16_0.png
	new file:   src/module_2/EDA_1_files/EDA_1_30_0.png
	new file:   src/module_2/EDA_1_files/EDA_1_33_1.png
	new file:   src/module_2/EDA_1_files/EDA_1_36_1.png
	new file:   src/module_2/EDA_1_files/EDA_1_37_1.png
	new file:   src/module_2/EDA_2.md
	new file:   src/module_2/EDA_2_files/EDA_2_22_0.png
	new file:   src/module_2/EDA_2_files/EDA_2_26_0.png
	new file:   src/module_2/EDA_2_files/EDA_2_30_0.png
	new file:   src/module_2/EDA_2_files/EDA_2_32_0.png
	new file:   src/module_2/EDA_2_files/EDA_2_9_0.png
